src/style.css

```diff
@@ -11,7 +11,17 @@ body {
   height: 100vh;
 }
 
-.container {
+/* .container{ */
+
+/* display: flex; */
+
+/* flex-direction: column; */
+
+/* align-items: flex-start; */
+
+/* } */
+
```

Comment:
[OPTIONAL] You have almost done with your project. Congratulations. But since you were giving the final touches, you forgot to remove the commented code. Kindly remove this to comply with the clean code practices.